src/main/java/uk/gov/register/presentation/resource/Pagination.java

```diff
@@ -4,13 +4,15 @@
 import javax.ws.rs.BadRequestException;
 import java.util.Optional;
 
-class Pagination {
+public class Pagination {
 
     private final long pageIndex;
+    private final String resourcePath;
     private final long totalEntries;
     private final long pageSize;
 
-    Pagination(Optional<Long> pageIndex, Optional<Long> pageSize, long totalEntries) {
+    Pagination(String resourcePath, Optional<Long> pageIndex, Optional<Long> pageSize, long totalEntries) {
```

Comment:
does this mean that the pagination may fail to render all pages if the estimate is wrong?